internal/distro/test/distro.go

```diff
@@ -0,0 +1,60 @@
+package test
```

Comment:
Is this package used anywhere? I replaced all usages with the fedoratest packages if I remember correctly.